package.json

```diff
@@ -3,9 +3,10 @@
   "type": "module",
   "version": "0.0.1",
   "description": "Utilities to have an async fetch function in Remix apps",
+  "packageManager": "pnpm@9.13.2",
   "engines": {
     "node": ">= 18",
-    "pnpm": ">= 9"
+    "pnpm": "9.13.2"
```

Comment:
I think we need to get rid of this altogether (it is saying "you can only use this library if you are using this version of `pnpm`"). We can leave the `packageManager` field though because that only applies when working on the project locally.